sim/paladin/soc.go

```diff
@@ -151,6 +151,6 @@ func (paladin *Paladin) registerSealOfCommand() {
 			},
 		})
 
-		paladin.spellsJoC[i] = judgeSpell
+		paladin.spellsJoC = append(paladin.spellsJoC, judgeSpell)
```

Comment:
Love this approach 👍 The only time I still use rank-indexed spells is if for some reason I need to specifically cast another spell of the same rank (e.g. shaman overloads)

Reply:
thanks!